src/locale.js

```diff
@@ -92,6 +92,37 @@ const locale = {
     'windy': 'Windy',
     'windy-variant': 'Windy'
   },
+  se: {
```

Comment:
According to [ISO 639-1](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=sv) the Language code should be `sv`.